packages/appstore/src/components/containers/trading-app-card.tsx

```diff
@@ -17,140 +16,150 @@ type TWalletsProps = {
+                                    size='xxxs'
+                                    line_height='s'
+                                    className='trading-app-card__details__short-code'
+                                >
+                                    {short_code_and_region}
+                                </Text>
+                            )}
+                        </div>
                         {is_wallet ? (
-                            <Text className='title' size='xs' line_height='s' color='prominent'>
-                                {is_wallet_demo && sub_title ? `${sub_title} ${localize('Demo')}` : sub_title}
+                            <Text
+                                className='title'
+                                size='xs'
+                                line_height='s'
+                                weight='bold'
+                                color={action_type === 'trade' ? 'prominent' : 'general'}
+                            >
+                                {is_wallet_demo && !sub_title && !is_deriv_platform
+                                    ? `${name} ${localize('Demo')}`
+                                    : name}
                             </Text>
                         ) : (
-                            <Text className='title' size='xs' line_height='s' color='prominent'>
-                                {!is_real && sub_title ? `${sub_title} ${localize('Demo')}` : sub_title}
-                            </Text>
-                        )}
-                        {!is_wallet && short_code_and_region && (
                             <Text
-                                weight='bolder'
-                                size='xxxs'
+                                className='title'
+                                size='xs'
                                 line_height='s'
-                                className='trading-app-card__details__short-code'
+                                weight='bold'
+                                color={action_type === 'trade' ? 'prominent' : 'general'}
                             >
-                                {short_code_and_region}
+                                {!is_real && !sub_title && !is_deriv_platform ? `${name} ${localize('Demo')}` : name}
                             </Text>
                         )}
-                    </div>
-                    {is_wallet ? (
-                        <Text
-                            className='title'
-                            size='xs'
-                            line_height='s'
-                            weight='bold'
-                            color={action_type === 'trade' ? 'prominent' : 'general'}
-                        >
-                            {is_wallet_demo && !sub_title && !is_deriv_platform ? `${name} ${localize('Demo')}` : name}
+                        <Text className='description' color={'general'} size='xxs' line_height='m'>
```

Comment:
```suggestion
                        <Text className='description' color={'general'} size='xxs'>
```
Line height is default to `m`